README.md

```diff
@@ -1100,6 +1100,24 @@ If you're already using Python 3.7, switch the `language_version` accordingly. F
 `stable` is a tag that is pinned to the latest release on PyPI. If you'd rather run on
 master, this is also an option.
 
+## GitHub actions
+
+Create a file named `.github/workflows/black.yml` inside your repository with:
```

Comment:
NIT: Might want to be clearer about 'black.yml' being the file name and the rest is the directory structure.

Reply:
Uhm... wouldn't know how to make it clearer without it being too verbose.

Reply:
Maybe this: "In the `.github/workflows` directory of your repository, create a file named `black.yml` with:"

Or is that too verbose in your opinion?

Reply:
I can't just copy-paste the whole path in one go now, can I?

Reply:
Makes sense.